### DISCUSSION TITLE
GitHub Pages Reveals Email and Password In The URL

### DISCUSSION THREAD

**Discussion by oriohac** [community/community#19485] [Projects and Issues] [answered]
When I want to share my GitHub Pages URL to whatsapp with the share button on Chrome, the URL exposes my email and password.

**Comment by melvincwng** [comment, marked as answer]
@oriohac, since this issue involves sensitive information/credentials, I suggest you to raise a ticket and directly report to [GitHub Bug Report](https://support.github.com/contact/bug-report).

Describe your issue over there and allow at least several working days for the support engineers to get back to you.

Hope you find the answer helpful. If helpful, please 'Mark as answer'.

Have a good day ahead 😊

**Reply by oriohac** [reply]
Thanks for your reply, I was looking for the bug report but never found it!

**Reply by melvincwng** [reply]
@oriohac, hey man, it's my pleasure in being able to assist in your inquiry :)

Do ping me here if you require any other assistance regarding this issue.

For other issues, please kindly create a new discussion thread instead (since this one is already addressed/marked as answer).

Thank you, and have a pleasant day ahead 👍🏻
